pontoon/base/static/js/main.js

```diff
@@ -513,6 +513,33 @@ var Pontoon = (function (my) {
         }).error(error).complete(complete);
       }
 
+      // Machine translation (Caighdean)
+      if (self.locale.code === 'ga-IE') {
+        requests++;
+
+        if (self.XHRCaighdeanMT) {
+          self.XHRCaighdeanMT.abort();
+        }
+
+        self.XHRCaighdeanMT = $.ajax({
+          url: '/machine-translation-caighdean/',
+          data: {
+	    path: $('#editor')[0].entity.path,
+	    key: $('#editor')[0].entity.key,
+            locale: self.locale.code
+          }
+        }).success(function(data) {
+          if (data.translation) {
+            append({
+              url: 'https://github.com/kscanne/caighdean',
+              title: 'Caighdean Machine Translation',
+              source: 'Caighdean Machine Translation',
+              translation: data.translation
```

Comment:
It would be useful to also add the `original` parameter here, so that we see what string was used as the input.

Reply:
i think the original would be the `gd` translation that is sent to the MT. We could add it in the response, but it wouldnt be used currently